apps/frontend/src/libs/components/chart/chart.tsx

```diff
@@ -6,11 +6,17 @@ import styles from "./styles.module.css";
 
 type Properties = {
 	data: ChartData;
+	isCursorPointer: boolean;
 };
 
-const Chart = ({ data }: Properties): JSX.Element => {
+const Chart = ({ data, isCursorPointer }: Properties): JSX.Element => {
 	return (
-		<LineChart data={data} height={40} width={120}>
+		<LineChart
+			data={data}
+			height={40}
+			style={{ cursor: isCursorPointer ? "pointer" : "default" }}
```

Comment:
It's better to style through the className

Reply:
I tried. It doesn't apply it this way.
Image from our dev route:
![image](https://github.com/user-attachments/assets/be14c347-ea12-4af5-85f2-673eae23b5d7)

Reply:
Oh, maybe then let's add `cursor: pointer` style to the `NavLink` with which you wrap chart?

Reply:
And leave this component without changes

Reply:
Tried this too. Styles applied to parent elements do not override or cascade down to child elements for the `cursor` property.